docs/guides/connecting_to_your_data/database/postgres.md

```diff
@@ -0,0 +1,146 @@
+---
+title: How to connect to your data in a postgresql database
+---
+import AddingCredentials from '../reference/database_credentials.md'
+import NextSteps from '../reference/link-to-validator.md'
+import ConnectionStringDetails from '../reference/configuration_explanation.md'
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+This guide will help you connect Great Expectations to data in a Postgresql database. We will be building a `Datasource` configuration, and using it to verify connectivity. By the end of this how-to guide, you should have a `Batch` representing a slice of data in your database, and a working code snippet that can be used as a starting point for more complex configurations.
```

Comment:
I find the "slice of your data" a bit awkward. No need to change until we iron out the Batch vs Validator messaging. The style guide also focuses on "you" language instead of "we" language.